formulas/wick-base/functions/wick-make-group

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env bash
+# Create or manage a group on the system.
+#
+# $1            - Groupname to create on the system
+#
+# If the given group already exists `groupadd` will not  be run.
+# This can be used in conjunction with wickMakeUser to assign
```

Comment:
Normally I would say to add backticks:

    ... with `wickMakeUser` to assign ...

However, I think this whole sentence should be stricken and instead `wickMakeUser` should automatically create groups when the group doesn't exist.

Reply:
Not sure if you got the point I was trying to make with the second sentence, so I shall reword. I don't think that you should say the user should use this in conjunction with `wickMakeUser` as that should be automatic. Maybe you could change the phrasing to say that `wickMakeUser` uses this function to manage groups.